packages/dashboard-v2/src/components/FileList/useFormattedFilesData.js

```diff
@@ -0,0 +1,26 @@
+import { useMemo } from "react";
+import prettyBytes from "pretty-bytes";
```

Comment:
Just a note of something I noticed, I usually try to name my imports either as official docs say or just camel case the name of the library like you did here so it's consistent across the codebase. I noticed in some past pull requests you used different names for the import of pretty-bytes and while it's not a problem or an issue, I like to keep the names of the imports consistent across the codebase and the above mentioned guidelines help me with that.

<img width="409" alt="Screenshot 2022-03-16 at 10 36 03" src="https://user-images.githubusercontent.com/3755029/158561223-b58a0dbf-3285-4e62-9e50-7ecc17b1eb31.png">

Feel free to resolve this comment once read, it's just a note.

Reply:
Good point, I'll make it consistent in the upcoming PR 👍